lib/rules/no-useless-rename.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
                     return;
                 }
 
-                if (properties[i].computed) {
+                if (properties[i].computed || !properties[i].key) {
```

Comment:
I think a comment here about why this guard is needed could be helpful for future contributors